### PR TITLE
lwimiq 08 15 back to basics

### PR DESCRIPTION
**Pull request by jntullo**
Back to basics with this week's edition of last week in manageiq 

cc: @JPrause 

👀 appreciated @syncrou @imtayadeway

### CODE REVIEW

File: site/_posts/2017-08-11-back-to-basics.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Zone selection has been improved in the UI to include [all zones](https://github.com/ManageIQ/manageiq-ui-classic/pull/1769), not just the zones in the current region. 
+
+### Update UI without Database Connection
+The task of updating the UI did not need a database connection to execute, but was still failing without one. An [update to that task](https://github.com/ManageIQ/manageiq-ui-classic/pull/1887)
+will allow `rake update:ui` to run without a database connection. 
+
+## Fixed
+
+### Removed assumptions from tests 
+Making assumptions in tests can lead to sporadic failures. By removing the assumption that the [region is 99](https://github.com/ManageIQ/manageiq/pull/15731) in tests, we improve our test reliability.
```
Comment (imtayadeway) [inline site/_posts/2017-08-11-back-to-basics.md]: I think this should say "that the region is not 99" (since it used to be 1, but is now random)

**Comment by JPrause** [comment]
LGTM 👍

File: site/_posts/2017-08-11-back-to-basics.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+### 31,494 LOC :scissors:
+This may be considered cheating as it's not a complete deletion, but this [PR](https://github.com/ManageIQ/manageiq/pull/15682) removed
+31,494 lines of code. Sure, they were added back to the new [manageiq-api repository](https://github.com/ManageIQ/manageiq-api), but it's still an impressive :scissors: feat!
+
+### Almost nothing
+From tens of thousands of lines deleted, to 0, [this](https://github.com/ManageIQ/manageiq-ui-classic/pull/1820) still counts! It's great to clean things up
+and remove unnecessary files.
+
+### Style Removals
+There were also some style removals, some for the [nav](https://github.com/ManageIQ/manageiq-ui-classic/pull/1803/files) and some others [here](https://github.com/ManageIQ/manageiq-ui-classic/pull/1802).
```
Comment (imtayadeway) [inline site/_posts/2017-08-11-back-to-basics.md]: Did you mean to link to the `/files` URI for one of these but not the other?

**Comment by imtayadeway** [comment]
🎉 LGTM! Great work @jntullo ! 🎈

File: site/_posts/2017-08-11-back-to-basics.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+### Zone Selection
+Zone selection has been improved in the UI to include [all zones](https://github.com/ManageIQ/manageiq-ui-classic/pull/1769), not just the zones in the current region. 
+
+### Update UI without Database Connection
+The task of updating the UI did not need a database connection to execute, but was still failing without one. An [update to that task](https://github.com/ManageIQ/manageiq-ui-classic/pull/1887)
+will allow `rake update:ui` to run without a database connection. 
+
+## Fixed
+
+### Removed assumptions from tests
```
Comment (syncrou) [inline site/_posts/2017-08-11-back-to-basics.md]: @jntullo - Extremely tiny nit - Possibly capitalize Assumptions and Tests to match the pattern of the other headings.

**Comment by JPrause** [comment]
@simaishi can you merge.

File: site/_posts/2017-08-11-back-to-basics.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Featured
+
+If you're users of or contributors to the API, you may be interested to hear that the API now has its own repository!
+Head on over to its new location [here](https://github.com/ManageIQ/manageiq-api) and check out the [talk](http://talk.manageiq.org/t/proposed-split-repo-manageiq-api/2587/3) 
+article for more information.
+
+## Improved
+
+### Zone Selection
+Zone selection has been improved in the UI to include [all zones](https://github.com/ManageIQ/manageiq-ui-classic/pull/1769), not just the zones in the current region.
```
Comment (bdunne) [inline site/_posts/2017-08-11-back-to-basics.md]: Based on the code change, it looks like it does the opposite.  Before it was showing `Zone.all` now it shows `Zone.in_my_region`

File: site/_posts/2017-08-11-back-to-basics.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Last Week in ManageIQ: Back to Basics"
+author: jntullo
+date: 2017-08-15
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! It's [Jillian](https://github.com/jntullo) again, here with the latest happenings the last *two* weeks in ManageIQ.
```
Comment (bdunne) [inline site/_posts/2017-08-11-back-to-basics.md]: > here with the latest happenings the last *two* weeks in ManageIQ

I feel like something may be missing here.
"here with the latest happenings (of / for / in) the last *two* weeks in ManageIQ"

**Comment by JPrause** [comment]
@bdunne looks like the last edit is in,..can you merge. Thanks!

**Pull request merged**